### ISSUE TITLE
Tutorial missions, selection rank and transport AI call are visible

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#2241]
The selection rank and transport AI unit states should be grey in the tutorial where the rest of the states are grey.

**Comment by sprunk** [comment]
Looks like these two are widgetspace and the lock only affects synced commands.

**Comment by GoogleFrog** [comment]
Yes, therein lies the difficulty. Additionally, selection rank should be fixed to its defaults in case someone accidentally changes them for the tutorial.

Widget space is actually a great place to store selection rank because different players can have different values for selection rank on the same unit.
